roles/openshift_gcp/defaults/main.yml

```diff
@@ -9,6 +9,7 @@ openshift_gcp_node_group_type: master
 openshift_gcp_ssh_private_key: ''
 
 openshift_gcp_project: ''
+openshift_gcp_network_project: "{{ openshift_gcp_project }}"
```

Comment:
This should best be moved below `openshift_gcp_network_name` and default to `''`